src/components/Personalization/createOnClickHandler.js

```diff
@@ -41,6 +41,8 @@ export default ({
             }
           };
         }
+        // TODO: move this in attachClickActivityCollector get link details here and determine whether should set use beacon
+        event.documentMayUnload(); // this is needed in case it is an exit link and but clickCollection is disabled
```

Comment:
I think its okay to just use /collect no matter what type of link it is.

Reply:
ok, thank you